README.md

```diff
@@ -343,7 +354,8 @@ const c = makeDomainFunction(z.object({ id: z.number() }))(async ({ id }) => Boo
 const results = await all(a, b, c)({ id: 1 })
 ```
 
-On the exemple above, the result will be of type `Result<[string, number, boolean]>`:
+For the exemple above, `data` will be of type `Result<[string, number, boolean]>`:
```

Comment:
This correction changed the meaning of the original sentence.
Data will be [string, number, boolean] but we want to show that the result of the application of the domain function is an object of type: Result<[string, number, boolean]> . Which might be an ErrorResult without data.

Reply:
My bad -- I had originally misunderstood the Result type, made similar changes in a few places, realized the mistake, reverted it everywhere else, but missed this one.